cirq-google/cirq_google/devices/grid_device.py

```diff
@@ -120,41 +187,64 @@ def _validate_device_specification(proto: v2.device_pb2.DeviceSpecification) ->
             raise ValueError("Invalid DeviceSpecification: target_ordering cannot be ASYMMETRIC.")
 
 
-def _build_gateset_and_gate_durations(
+def _serialize_gateset_and_gate_durations(
+    out: v2.device_pb2.DeviceSpecification,
+    gateset: cirq.Gateset,
+    gate_durations: Mapping['cirq.GateFamily', 'cirq.Duration'],
```

Comment:
nit: don't need to quote `cirq` types since we're in `cirq_google`:
```suggestion
    gate_durations: Mapping[cirq.GateFamily, cirq.Duration],
```